miden-lib/asm/wallet/basic.masm

```diff
@@ -0,0 +1,30 @@
+use.miden::sat::account
+use.miden::sat::tx
+use.miden::eoa::basic_auth
+
+#! Stack: [ASSET]
+#! Output: [ASSET']
+#!
+#! - ASSET' final asset in the account vault defined as follows:
+#!   - If ASSET is a non-fungible asset, then ASSET' is the same as ASSET.
+#!   - If ASSET is a fungible asset, then ASSET' is the total fungible asset in the account vault
+#!     after ASSET was added to it.
+export.receive_asset
```

Comment:
As mentioned in the other comment, I think returning `ASSET'` may leak some info. So, we should probably drop `ASSET'` from the stack and replace it with 0s (will take extra 8 cycles). This way, the input/output would look like:

```
Inputs:  [ASSET, ...]
Outputs: [0, 0, 0, 0, ...]
```

Also, a couple of nits:
- I would use the above format for showing inputs/outputs of the stack.
- We should add a single sentence at the top comment to describe the purpose of the procedure.